packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -74,7 +74,7 @@ const Skeleton = React.forwardRef(function Skeleton(props, ref) {
     animation = 'pulse',
     classes,
     className,
-    component: Component = 'div',
+    component: Component = ['rect', 'circle'].includes(props.variant) ? 'div' : 'span',
```

Comment:
IE 11

Reply:
so change to:
`['rect', 'circle'].indexOf(props.variant) > -1 ? 'div' : 'span'`
and as i see i also have to update the test case to 'span'.
should i made another commit or change squash the commit?

Reply:
indexOf sounds great :). As you prefer for the commit.